server/channels/store/store.go

```diff
@@ -234,6 +234,9 @@ type ChannelStore interface {
 	UpdateMemberNotifyProps(channelID, userID string, props map[string]string) (*model.ChannelMember, error)
 	GetMembers(channelID string, offset, limit int) (model.ChannelMembers, error)
 	GetMember(ctx context.Context, channelID string, userID string) (*model.ChannelMember, error)
+	// GetMemberOnly is a lite version of GetMember where it does not join
+	// with the different schemes tables.
+	GetMemberOnly(ctx context.Context, channelID string, userID string) (*model.ChannelMember, error)
```

Comment:
If the scheme is the main difference, should we name this `GetMemberWithoutSchemeRoles`? Too long maybe. Ugh, naming is difficult. The comment is good, by the way, thanks for that!